nodepool/files/nodepool.yml

```diff
@@ -24,6 +24,17 @@ labels:
     min-ready: 0
     max-ready-age: 3600
 
+  # rpco snapshots
+  - name: rpc-r14-xenial_loose_artifacts-swift-7-15
```

Comment:
nit: These labels are very long, and the ``-7-15`` could be dropped, I think, given that this label can never use another flavor.